includes/updates/update-1.9.1.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * Alter some `option_name` value in `wp_options` table
+ *
+ * It's caused for moving the CRM email connectivity to Email Section
+ */
+function erp_settings_alter_options_table_crm_email_1_9_1() {
+    global $wpdb;
+
+    // Check if someone already added the new option_names, Then just delete the previous one or update
+    $email_gmail_connect_exists = $wpdb->get_row(
+        $wpdb->prepare(
+            "SELECT * FROM {$wpdb->prefix}options WHERE option_name = %s", [ 'erp_settings_erp-email_gmail' ]
+        )
+    );
+
+    // Delete or Update `erp_settings_erp-crm_email_connect_gmail` to `erp_settings_erp-email_gmail`
+    if ( empty( $email_gmail_connect_exists ) ) {
+        $wpdb->query( "UPDATE `{$wpdb->prefix}options` SET `option_name` = 'erp_settings_erp-email_gmail' WHERE `option_name`= 'erp_settings_erp-crm_email_connect_gmail';" );
+    } else {
+        delete_option( 'erp_settings_erp-crm_email_connect_gmail' );
+    }
```

Comment:
You could just avoid the whole process by doing the following. Though the internal process is a bit lengthy, it is safer. Also, as it will run just one time, the lengthy process could be considered.

```
$gmail_connect_old  = get_option( 'erp_settings_erp-crm_email_connect_gmail' );
$gmail_connect_new = get_option( 'erp_settings_erp-email_gmail', false );

if ( false === $gmail_connect_new && ! empty( $gmail_connect_old ) ) ) {
    update_option( 'erp_settings_erp-email_gmail', $gmail_connect_old );
}

delete_option( 'erp_settings_erp-crm_email_connect_gmail' );
```